root/static/scripts/edit/URLCleanup.js

```diff
@@ -5293,6 +5338,39 @@ const CLEANUPS: CleanupEntries = {
       return {result: isAProfile, target: ERROR_TARGETS.URL};
     },
   },
+  'ticketmaster': {
+    match: [new RegExp('^(https?://)?(www\\.)?ticketmaster\\.com', 'i')],
+    restrict: [LINK_TYPES.ticketing],
+    clean: function (url) {
+      url = url.replace(/^(?:https?:\/\/)?(?:www\.)?ticketmaster\.com\/(?:[\w-]+\/)?(artist|event|venue)\/([0-9A-F]+).*$/, 'https://www.ticketmaster.com/$1/$2');
```

Comment:
Wasn’t there some redirect happening when removing the slug name from Ticketmaster URLs?

Reply:
I think what you're thinking about is the redirect when moving from `concerts.livenation` to `www.livenation` :)

Reply:
No, it was about the examples you shared on IRC.

Reply:
Yeah, but those were event examples for livenation IIRC, not ticketmaster ones?

Reply:
Yeah, but those were event examples for livenation IIRC, not ticketmaster ones?

Reply:
Correct.